dataplane/pkg/common/egress_interface.go

```diff
@@ -78,25 +78,36 @@ func parseProcFile(scanner *bufio.Scanner) (string, net.IP, error) {
 		if strings.TrimSpace(parts[1]) == "00000000" {
 			outgoingInterface := strings.TrimSpace(parts[0])
 			defaultGateway := strings.TrimSpace(parts[2])
-			ip := parseGatewayIP(defaultGateway)
+			ip, err := parseGatewayIP(defaultGateway)
+			if err != nil {
+				logrus.Printf("Error Processing gateway IP %v for outgoing interface: %v", defaultGateway, outgoingInterface)
```

Comment:
Not something crucial, but there's a convention in golang to write log messages in lower case.

Reply:
it is related to errors